src/List/list.store.js

```diff
@@ -95,11 +95,29 @@ export default Store.create({
         this.listSourceSubject.onNext(Observable.fromPromise(this.state.pager.getPreviousPage()));
     },
 
+    buildPager(listSearchPromiseWithPagination, listSearchPromiseWithAllUsers) {
+        return Promise.all([listSearchPromiseWithPagination, listSearchPromiseWithAllUsers]).then(
+            ([model, model1]) => {
+                model.pager.total = model1.valuesContainerMap.size;
+                return model;
+            }
+        );
+    },
     filter(options, complete, error) {
         getD2().then(d2 => {
             const { filters, ...listOptions } = options;
-            const listSearchPromise = getList(d2, filters, listOptions);
-            this.listSourceSubject.onNext(Observable.fromPromise(listSearchPromise));
+            const listSearchPromiseWithPagination = getList(d2, filters, listOptions);
+            const listSearchPromiseWithAllUsers = getList(d2, filters, {
+                ...listOptions,
+                paging: false,
```

Comment:
As we only need the users count for this second request, I'd ask for fewer fields. This should work here: `fields: "id",`